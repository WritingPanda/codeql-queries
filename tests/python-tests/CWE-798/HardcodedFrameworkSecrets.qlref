CWE-798/HardcodedFrameworkSecrets.ql