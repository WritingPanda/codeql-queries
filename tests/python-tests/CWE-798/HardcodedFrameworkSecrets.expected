edges
| hardcoded_secrets.py:49:5:49:18 | ControlFlowNode for Str | hardcoded_secrets.py:50:72:50:72 | ControlFlowNode for w |
| settings.py:12:17:12:35 | ControlFlowNode for Str | settings.py:13:14:13:26 | ControlFlowNode for RANDOM_STRING |
nodes
| hardcoded_secrets.py:10:18:10:29 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:11:28:11:39 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:12:30:12:41 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:22:30:22:47 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:28:38:28:55 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:34:53:34:69 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:35:50:35:66 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:40:47:40:59 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:41:38:41:50 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:47:72:47:85 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:49:5:49:18 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:50:72:50:72 | ControlFlowNode for w | semmle.label | ControlFlowNode for w |
| hardcoded_secrets.py:70:23:70:40 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:71:27:71:48 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| hardcoded_secrets.py:72:23:72:42 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| settings.py:5:14:5:29 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| settings.py:12:17:12:35 | ControlFlowNode for Str | semmle.label | ControlFlowNode for Str |
| settings.py:13:14:13:26 | ControlFlowNode for RANDOM_STRING | semmle.label | ControlFlowNode for RANDOM_STRING |
subpaths
#select
| hardcoded_secrets.py:10:18:10:29 | ControlFlowNode for Str | hardcoded_secrets.py:10:18:10:29 | ControlFlowNode for Str | hardcoded_secrets.py:10:18:10:29 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:10:18:10:29 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:11:28:11:39 | ControlFlowNode for Str | hardcoded_secrets.py:11:28:11:39 | ControlFlowNode for Str | hardcoded_secrets.py:11:28:11:39 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:11:28:11:39 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:12:30:12:41 | ControlFlowNode for Str | hardcoded_secrets.py:12:30:12:41 | ControlFlowNode for Str | hardcoded_secrets.py:12:30:12:41 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:12:30:12:41 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:22:30:22:47 | ControlFlowNode for Str | hardcoded_secrets.py:22:30:22:47 | ControlFlowNode for Str | hardcoded_secrets.py:22:30:22:47 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:22:30:22:47 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:28:38:28:55 | ControlFlowNode for Str | hardcoded_secrets.py:28:38:28:55 | ControlFlowNode for Str | hardcoded_secrets.py:28:38:28:55 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:28:38:28:55 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:34:53:34:69 | ControlFlowNode for Str | hardcoded_secrets.py:34:53:34:69 | ControlFlowNode for Str | hardcoded_secrets.py:34:53:34:69 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:34:53:34:69 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:35:50:35:66 | ControlFlowNode for Str | hardcoded_secrets.py:35:50:35:66 | ControlFlowNode for Str | hardcoded_secrets.py:35:50:35:66 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:35:50:35:66 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:40:47:40:59 | ControlFlowNode for Str | hardcoded_secrets.py:40:47:40:59 | ControlFlowNode for Str | hardcoded_secrets.py:40:47:40:59 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:40:47:40:59 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:41:38:41:50 | ControlFlowNode for Str | hardcoded_secrets.py:41:38:41:50 | ControlFlowNode for Str | hardcoded_secrets.py:41:38:41:50 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:41:38:41:50 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:47:72:47:85 | ControlFlowNode for Str | hardcoded_secrets.py:47:72:47:85 | ControlFlowNode for Str | hardcoded_secrets.py:47:72:47:85 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:47:72:47:85 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:50:72:50:72 | ControlFlowNode for w | hardcoded_secrets.py:49:5:49:18 | ControlFlowNode for Str | hardcoded_secrets.py:50:72:50:72 | ControlFlowNode for w | Use of $@. | hardcoded_secrets.py:49:5:49:18 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:70:23:70:40 | ControlFlowNode for Str | hardcoded_secrets.py:70:23:70:40 | ControlFlowNode for Str | hardcoded_secrets.py:70:23:70:40 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:70:23:70:40 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:71:27:71:48 | ControlFlowNode for Str | hardcoded_secrets.py:71:27:71:48 | ControlFlowNode for Str | hardcoded_secrets.py:71:27:71:48 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:71:27:71:48 | ControlFlowNode for Str | hardcoded credentials |
| hardcoded_secrets.py:72:23:72:42 | ControlFlowNode for Str | hardcoded_secrets.py:72:23:72:42 | ControlFlowNode for Str | hardcoded_secrets.py:72:23:72:42 | ControlFlowNode for Str | Use of $@. | hardcoded_secrets.py:72:23:72:42 | ControlFlowNode for Str | hardcoded credentials |
| settings.py:5:14:5:29 | ControlFlowNode for Str | settings.py:5:14:5:29 | ControlFlowNode for Str | settings.py:5:14:5:29 | ControlFlowNode for Str | Use of $@. | settings.py:5:14:5:29 | ControlFlowNode for Str | hardcoded credentials |
| settings.py:13:14:13:26 | ControlFlowNode for RANDOM_STRING | settings.py:12:17:12:35 | ControlFlowNode for Str | settings.py:13:14:13:26 | ControlFlowNode for RANDOM_STRING | Use of $@. | settings.py:12:17:12:35 | ControlFlowNode for Str | hardcoded credentials |